### ISSUE TITLE
Permission to comment on PR 27235

### ISSUE THREAD

**Issue by 1440000bytes** [bitcoin/bitcoin#27243]
### Motivation

I do not understand the reason for being restricted as new github accounts can comment on it

**Comment by fanquake** [comment]
This must be an issue with your account, as commenting is not restricted in that PR. i.e You may be blocked on GitHub by the PR author, which would prevent you from interacting with them. 

That is not something that can be resolved by the project. I'm going to close this issue, however feel free to continue discussion.

**Comment by 1440000bytes** [comment]
@fanquake

> That is not something that can be resolved by the project. I'm going to close this issue, however feel free to continue discussion.

This has to be resolved by the project else authors can always block people they dont want to review in their pull requests.

**Comment by achow101** [comment]
> This has to be resolved by the project else authors can always block people they dont want to review in their pull requests.

Individuals are free to block whoever they wish, the project cannot force anyone to do anything. Github does not allow us to override this, the project cannot do anything to change that.

**Comment by 1440000bytes** [comment]
> > This has to be resolved by the project else authors can always block people they dont want to review in their pull requests.
> 
> Individuals are free to block whoever they wish, the project cannot force anyone to do anything. Github does not allow us to override this, the project cannot do anything to change that.

What if I share example in which project has done things to override this? Is this a joke?

Can I open a pull request and block everyone? Is this the practice you are trying to suggest?

**Comment by achow101** [comment]
> What if I share example in which project has done things to override this?

Please do. I am not aware of any feature that allows us to override it.

However I am aware of a few instances where contributors have unblocked others after they realized that the block was preventing those people from commenting in their PRs, but these were done by those people voluntarily, not by the project.

> Can I open a pull request and block everyone?

You are free to do that, although it will likely be closed if no one is able to post a review.

**Comment by earonesty** [comment]
if you want to comment, open a new PR with the same changes.

**Comment by 1440000bytes** [comment]
I will get back to this drama in a few days. I have flight in a few hours.

Only 2 things that I wanted to comment in the CENSORED pull request of BITCOIN CORE:

1. Steps to reproduce the issue?

2. https://github.com/bitcoin/bitcoin/pull/27050 is not likely to get merged which was mentioned in [description](https://i.imgur.com/iPkMtm6.png) 

3. If this is an issue, 6 devs reviewed https://github.com/bitcoin/bitcoin/pull/25315. Do we have any plans to avoid this?

If nothing works. I think I have the resolution seeing the attitude of present maintainers and how they want to maintain this project:

![image](https://user-images.githubusercontent.com/94559964/224506767-26036b84-dbd3-4cd3-9e8a-5ca8da2338e4.png)

**Comment by dergoegge** [comment]
There really is no drama for you to get back to here. I blocked you and I will not be unblocking you. You have a habit of posting rude and spammy comments which I find incredibly annoying. In my opinion, some of the things you have said in the past would warrant a permanent ban.

**Comment by earonesty** [comment]
and yet he's perfectly free to open up a new PR with the same set of changes and comments all over it...

censorship doesn't exist

**Comment by 1440000bytes** [comment]
> and yet he's perfectly free to open up a new PR with the same set of changes and comments all over it...
> 
> censorship doesn't exist

Thanks for the suggestion. I will follow these in the future for PRs trying to avoid reviews:

1. Open a PR with same commit and add comments
2. If 1 doesn't work. Send email to bitcoin-core-dev@lists.linuxfoundation.org
3. If 2 doesn't work. Use an invite only [service](https://i.imgur.com/HUGS3gG.png) to post anonymous reviews
